packages/agoric-cli/src/chain-config.js

```diff
@@ -236,7 +236,6 @@ export function finishCosmosGenesis({ genesisJson, exportedGenesisJson }) {
 
   // Set the denomination for different modules.
   genesis.app_state.mint.params.mint_denom = MINT_DENOM;
-  genesis.app_state.crisis.constant_fee.denom = MINT_DENOM;
```

Comment:
Does this run afoul of requiring a bundle update?

Reply:
Nope, the CLI does not impact bundles. Because of cycles in our dependency graph it will cause a version bump of some dependencies, but other PRs will do that too.